sdk/samples/router/general.py

```diff
@@ -200,6 +200,13 @@ def main(turing_api: str, project: str):
             id="fee-fi-fo-fum", endpoint="http://fox-says.fee-fi-fo-fum", timeout="20ms"
         )
     )
+    my_router_config.rules.append(
+        TrafficRule(
+            name="giant-rule",
+            conditions=[HeaderTrafficRuleCondition(field="name", values=["giant"])],
+            routes=["fee-fi-fo-fum"],
+        ),
+    )
```

Comment:
Added this traffic rule because the sample no longer works (a change was made to ensure all routes of a router with traffic rules must be included in at least 1 traffic rule).